### PR TITLE
Adding support for MCP3008

### PR DESCRIPTION
**Pull request by maglims**

### CODE REVIEW

File: PotentiometerSensor/CS/MainPage.xaml.cs [hunk 1]
```diff
@@ -192,12 +203,12 @@ private void MainPage_Unloaded(object sender, object args)
                 ledPin.Dispose();
             }
         }
-        enum AdcDevice { NONE, MCP3002, MCP3208 };
+        enum AdcDevice { NONE, MCP3002, MCP3208, MCP3008};
 
         /* Important! Change this to either AdcDevice.MCP3002 or AdcDevice.MCP3208 depending on which ADC you chose     */
```
Comment (apdutta) [inline PotentiometerSensor/CS/MainPage.xaml.cs]: Need to add MCP3008 to the comment

**Comment by apdutta** [comment]
Looks good to me :+1:

**Comment by turkycat** [comment]
Have you updated the WOD content to show support for the 3008 chip as well as an additional step to verify the pin number is correct depending on which board you are using?

**Comment by maglims** [comment]
Apu did the wod update, with the mcp3008 and also fritzing

**Comment by turkycat** [comment]
I'll sign off, 2/2

**Pull request merged**